backend/middleware/sanitize.test.ts

```
@@ -141,21 +141,15 @@
-      ((result['b'] as Array<unknown>)[1] as Record<string, unknown>)['c']
-    ).toContain('<i>ok</i>');
-    expect(
-      ((result['b'] as Array<unknown>)[1] as Record<string, unknown>)['d']
-    ).toBe('evil()');
-    expect((result['e'] as Record<string, unknown>)['f']).toBe('plain');
-    expect((result['e'] as Record<string, unknown>)['g']).toContain(
-      '<b>rich</b>'
-    );
-    expect(result['h']).toBe('should skip');
+    const result = sanitizeWithSchema(input, schema);
+    expect((result as any).a).toContain('<b>Safe</b>');
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)